armtemplates/template.bicep

```diff
@@ -0,0 +1,280 @@
+param virtualMachines_iagora_name string = 'iagora'
+param publicIPAddresses_iagora_ip_name string = 'iagora-ip'
+param virtualNetworks_iagora_vnet_name string = 'iagora-vnet'
+param networkInterfaces_iagora731_z1_name string = 'iagora731_z1'
+param networkSecurityGroups_iagora_nsg_name string = 'iagora-nsg'
+param sshPublicKey string 
+param vmSize string = 'Standard_B2s' 
+
+//NSG ressourc
```

Comment:
What is NSG? Wouldn't this comment be easier to understand if we spelled it out instead of using an acronym?

Reply:
Yes, it's the best practise !!
It's mean Network Security Group 
it's just a comment to explain that i started nsg configuration's parameter

Reply:
OK) Great) Could you then update this comment to:
```
// Network Security Group Ressource
```
?